docs/guides/deploy-k8s.md

```diff
@@ -27,6 +27,118 @@ This guide explains how to deploy the Sensitive Data Archive (SDA) in kubernetes
 
 ## Minimal working configuration
 
+## Minimal set of configuration variables
+
+Here we provide a minimal list of variables that need to be configured in the `values.yml` file of the Helm charts in order to achieve a working deployment of the `sensitive data archive` services. In the following it is assumed that a *federated* setup is being deployed.
```

Comment:
maybe we can link to the files for each chart ? when the https://github.com/neicnordic/sensitive-data-archive/pull/398 is merged